src/Dscp/Educator/Web/Student/Handlers.hs

```diff
@@ -28,54 +32,77 @@ servantHandlers
     :<|> deleteSubmission
     :<|> getProofs
 
-notImplemented :: MonadIO m => m a
-notImplemented = UIO.throwIO $ err500 { errBody = "Not implemented" }
+student :: Student
+student = Core.mkAddr . toPublic $ deterministic "" genSecretKey
+
+{- GRAND TODO LIST
+
+1. Student authentication ([DSCP-141]).
+
+2. Errors:
+2.1. Catch and rethrow SQL DomainErrors?
```

Comment:
Would be nice indeed. Also, the error response should probably match the format in the spec)